casper/src/main/scala/coop/rchain/casper/storage/RNodeKeyValueStoreManager.scala

```diff
@@ -20,9 +20,9 @@ object RNodeKeyValueStoreManager {
   private val evalHistoryEnvConfig = LmdbEnvConfig(name = "eval/history", maxEnvSize = 1 * tb)
   private val evalColdEnvConfig    = LmdbEnvConfig(name = "eval/cold", maxEnvSize = 1 * tb)
   // Blocks
-  private val blockStorageEnvConfig  = LmdbEnvConfig(name = "blockstorage", maxEnvSize = 1 * tb)
-  private val dagStorageEnvConfig    = LmdbEnvConfig(name = "dagstorage", maxEnvSize = 100 * gb)
-  private val deployStorageEnvConfig = LmdbEnvConfig(name = "deploystorage", maxEnvSize = 1 * gb)
+  private val blockStorageEnvConfig = LmdbEnvConfig(name = "blockstorage", maxEnvSize = 1 * tb)
+  private val dagStorageEnvConfig   = LmdbEnvConfig(name = "dagstorage", maxEnvSize = 100 * gb)
+  private val deployPoolEnvConfig   = LmdbEnvConfig(name = "deploypool", maxEnvSize = 1 * gb)
```

Comment:
Why not call deploy DB file in the same way as other files?

Reply:
IMO `deploystorage` tells that this is a general storage for all deploys, not a pool of deploys submitted to validator.  I would expect that `deploystorage` is a storage of deploys coming with blocks.

Reply:
Reamed to `deploypoolstorage` in [Rename pooled](https://github.com/rchain/rchain/pull/3673/commits/b4b4ab17ede42f7d141717aa32d0a9b9cd246d1a)